modules/ROOT/pages/migration/index.adoc

```diff
@@ -20,6 +20,42 @@ npm update @neo4j/graphql
 
 Here is a list of all the breaking changes from version 5.0.0 to 6.0.0.
 
+=== The deprecated `_NOT` filters are no longer supported
+
+The following deprecated `NOT` filters are removed from the schema since they are no longer supported:
+
+  - `_NOT`
+  - `_NOT_CONTAINS`
+  - `_NOT_ENDS_WITH`
+  - `_NOT_IN`
+  - `_NOT_STARTS_WITH`
+  - `_NOT_INCUDES`
```

Comment:
I had an extra look I noticed that we also had the `_NOT` applied to `node_NOT`, `edge_NOT`,  `actors_NOT`, and `actorsConnection_NOT` - what is the general recommendation here be specific and verbose with different examples for different cases, or it's enough clear that all the `_NOT` are removed in favor of the same alternative? 

@mjfwebb @rsill-neo4j

Reply:
My personal opinion is explicitness trumps implicitness in documentation, but there's probably a balance to be had there. I think so long as you do mention them then it's valuable, though maybe not requiring full examples.

Reply:
Sounds good, I put the `@relationship` filter as separate case as they have a different alternative!